lib/client.js

```diff
@@ -166,29 +170,44 @@
       string = string.replace(/\n /,"\n");
       return string;
     }
-
+    
     function assembleResponse (response) {
         var sender,
             formatUserhost,
             formatNick,
             formattedReturn,
+			host,
+			formatHost,
+			shost,
             nick;
```

Comment:
Since the rest of the file is using spaces instead of tabs, please correct this to use spaces to keep consistency.